homeassistant/components/somfy/.translations/en.json

```diff
@@ -0,0 +1,24 @@
+{
+    "config": {
+        "abort": {
+            "already_setup": "You can only configure one Somfy account.",
+            "authorize_url_fail": "Unknown error generating an authorize url.",
+            "authorize_url_timeout": "Timeout generating authorize url.",
+            "external_setup": "Somfy successfully configured from another flow.",
+            "no_flows": "You need to configure Somfy before being able to authenticate with it. [Please read the instructions](https://www.home-assistant.io/components/somfy/)."
+        },
+        "create_entry": {
+            "default": "Successfully authenticated with Somfy."
+        },
+        "error": {
+            "follow_link": "Please follow the link and authenticate before pressing Submit"
+        },
+        "step": {
+            "auth": {
+                "description": "Please follow this [Link]({authorization_url}) and <b>Accept</b> access to your Somfy account, then come back and press <b>Submit</b> below",
```

Comment:
You can't use HTML

Reply:
If I remove the link, I break my component. Indeed, Somfy the Oauth protocol. So I need end user be redirect to the somfy portal. You already suggest me to use a draft you created for Nest: https://github.com/home-assistant/home-assistant/pull/19548#discussion_r261866116
But as you can see in my comments, I don't know how to use it.

Reply:
I think we just need to remove the html tags.

Reply:
In the comment just above @balloob does not want markdown too.

Reply:
Yes, no markdown or html.

Reply:
hmm I see that we have a few instances for markdown, markdown is ok for now.

Reply:
Great. You save my component!